docs/api/code_check.rst

```diff
@@ -406,3 +418,52 @@ created target is added to the parent `check` build target. This target will not
 which files do not conform to your style guide.
 .. Note::
   Setting MODIFY_FILES to FALSE is only supported in Uncrustify v0.61 or greater.
+
+
+blt_add_cmakeformat_target
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: cmake
+
+    blt_add_cmakeformat_target( NAME              <Created Target Name>
+                           MODIFY_FILES      [TRUE | FALSE (default)]
```

Comment:
nitpick: line these up please.

Reply:
Done in [00dc40c0](https://github.com/LLNL/blt/pull/415/commits/00dc40c0ae97d4b2c157fa633d13e5c58dcefa70).